packages/server/tests/clients/servicesClient.ts

```diff
@@ -275,6 +275,13 @@ export default class ServicesClient {
         const receipt = await response.getReceipt(this.client);
         this.logger.info(`File ${fileId} updated with status: ${receipt.status.toString()}`);
     }
+
+    async getAccountBalanceInWeiBars(account: string | AccountId) {
+        const balance = await this.executeQuery(new AccountBalanceQuery()
```

Comment:
Pull out  `AccountBalanceQuery` execution into a separate method as it's used multiple times in this class
Will need to update it to handle `AccountId` also
```suggestion
        const balance = await this.getAccountBalance(account)
```

e.g.
```
    async getAccountBalance(account: string | AccountId): Promise<AccountBalance> {
        return this.executeQuery(new AccountBalanceQuery()
            .setAccountId(AccountId.fromString(account)), this.clientMain);
    }
```